pipeline/tasks/task-scripts/activate-service-account.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set +x
```

Comment:
Do we really need `set -x` here? This'll log out keys in the build logs, is that necessary?
I would suggest setting the break–on–any–error flag instead: `set -e`

Reply:
We've got set +x which leads to it NOT being echo'd in the build logs.  Will chat to confirm it's OK.

I'll add set -e on all build scripts

Reply:
Yes, it's `set +x`, my mistake